scripts/setup-helper-functions.sh

```diff
@@ -20,6 +20,14 @@ DEPENDENCY_DIR=${DEPENDENCY_DIR:-$(pwd)/deps-download}
 OS_CXXFLAGS=""
 NPROC=${BUILD_THREADS:-$(getconf _NPROCESSORS_ONLN)}
 
+if [[ "${VERBOSE_LOGGING}" == "true" ]]; then
```

Comment:
Same here. Let's define `CURL_OPTIONS` and `CMAKE_OPTIONS` and leave the value to the user to set.